Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -197,6 +198,11 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
         onBackPressed()
     }
 
+    fun navigateToContactDiaryActivity() {
+        ContactDiaryActivity.start(this)
+        finish()
```

Comment:
Shouldn't the user get back the to main CWA activity after exiting the diary?

Reply:
Ah yes you're right :) I forgot to remove this before pushing.